README.md

```diff
@@ -1,2 +1,60 @@
-# comp110-worksheet-5
-Base repository for COMP110 worksheet 5
+# COMP110-Worksheet-5
+
+## Joseph Spaul
+
+
+## (a)
+
+The task that this algorithm performs is for it to compare the length of two lists (list[i] and list[j]) and the 
+compare the values inside the lists. If the values of one list are not equal, but the length of the two lists are
+equal, the algorithm will return True. If the values are equal and the length of the lists are equal, then the 
+algorithm will return False. The same can be said for if the values of i and j are not equal and the length of 
+the lists are not equal too.
```

Comment:
Actually there is only one list -- the algorithm checks if it has any duplicated elements